webpack.config.js

```diff
@@ -4,14 +4,14 @@ module.exports = {
     filename: 'dist/bundle.js'
   },
   resolve: {
-    extensions: ['.ts', '.js', '.tsx', '.jsx', '']
+    extensions: ['.ts', '.js', '.tsx', '.jsx']
   },
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.tsx?$/,
         exclude: /node_modules/,
-        loader: 'ts-loader'
+        loader: 'awesome-typescript-loader'
```

Comment:
One thing that I like from aweseom-typescript-loader is the flag "useBabel: true" TS >> ES6 >> babel >> ES5, what do you think about this?